### ISSUE TITLE
Change Weekly Directory 

### ISSUE THREAD

**Issue by andrewenriquez** [evilhero/mylar#2318]
Hello, I'm having an issue changing the folder where the Weekly comics are being created. I love the feature but I'd like to move it from /comics to /comics/Weekly as I feel it will unclutter my main folder. 

I went into the config.ini and changed some variables. 
![image](https://user-images.githubusercontent.com/35500163/62247717-40348f80-b39b-11e9-963f-1b944c43a719.png)

But that didn't seem to take effect. I can see in the config listing it's not changed. 
![image](https://user-images.githubusercontent.com/35500163/62247810-783bd280-b39b-11e9-8d44-92c5d3f86a34.png)

Is it possible to change the directory from the UI because I can't seem to figure out how. Also, in the config directory, I notice a third variable 'weekfolder_format' that is not in my config.ini file when I edit it. This makes me believe the config.ini is in a different location? I'm using Mylar docker on Unraid 6.6.6.

Any help would be appreciated, thanks.

**Comment by barbequesauce** [comment]
This might sound silly, but just to be sure: you’re not changing the ini while Mylar is running, are you? It overwrites the file on shutdown...

Otherwise - the link you clicked to get to that screen you pasted above is the path to the loaded ini file. If the file at that location has different contents, there is likely some docker data director mapping wizardry in play; we’ve seen that before with LSIO docker files among others. Check with the source of the container...

**Comment by andrewenriquez** [comment]
Good point. Let me try that.

**Comment by andrewenriquez** [comment]
> Good point. Let me try that.

Tried shutting down and editing the .ini file but still don't get the changes to stick.

**Comment by barbequesauce** [comment]
“Don’t stick” - they don’t show in the config dump screen?

Probably best if you inquire with whoever rolled your docker container... happy to join in the conversation if needed, just pass along the issue number or irc/discord info...

**Comment by evilhero** [comment]
Whatever docker version you're running is seriously out of date with mylar. Your ini file should have sections within based on categories (ie. [General], [Post-processing], etc). The fact that your quick paste above of the ini shows no sections where there should be (as well as the 0/1 not being used anymore in favour of True/False), would probably indicate that you're editing the wrong ini file.

In the 1st tab of the mylar  configuration screen, it will indicate the location where the ini resides that's being used - I'd wager to guess that's different than the file that you're trying to edit.

**Comment by andrewenriquez** [comment]
![image](https://user-images.githubusercontent.com/35500163/62263545-7ee53c80-b3d1-11e9-9ded-f997860a5600.png)

Here's that first tab of the mylar configuration screen. 

![image](https://user-images.githubusercontent.com/35500163/62263789-50b42c80-b3d2-11e9-978b-cef0f3b1d4ea.png)

Here's the docker path showing the config file is mapped to my appdata folder. That's where the config.ini is located and the one i'm trying to edit. I can't seem to find a seperate config.ini file anywhere on my server.

**Comment by andrewenriquez** [comment]
Okay, this is embarrassing but you guys were right. I was editing the wrong config.ini file. I guess I had 2 mylar config folders created when I downloaded a different mylar docker and I never deleted that folder when I removed that docker image. 

![image](https://user-images.githubusercontent.com/35500163/62264333-5448b300-b3d4-11e9-93bc-8ef9b0d24918.png)

Here's it working just how I want it. Thanks.
